gwcs/wcs.py

```diff
@@ -1385,20 +1385,20 @@ def to_fits_sip(self, bounding_box=None, max_pix_error=0.25, degree=None,
             Maximum allowed error over the domain of the pixel array. This
             error is the equivalent pixel error that corresponds to the maximum
             error in the output coordinate resulting from the fit based on
-            a nominal plate scale.
+            a nominal plate scale. Must be greater than zero.
         degree : int, optional
-            Degree of the SIP polynomial. If supplied, max_pixel_error is ignored.
-        max_inv_error : float, optional
+            Degree of the SIP polynomial.
+        max_inv_pix_error : float, optional
             Maximum allowed inverse error over the domain of the pixel array
-            in pixel units. If None, no inverse is generated.
+            in pixel units. Must be greater than zero.
         inv_degree : int, optional
-            Degree of the inverse SIP polynomial. If supplied max_inv_pixel_error
-            is ignored.
+            Degree of the inverse SIP polynomial. If less than 0, no inverse
```

Comment:
I do not think specifying a degree of 0 would work. Looking at `_fit_2D_poly` it seems that a degree of 0 would result in undefined values.

Reply:
So, please, let's use `0` (and even `None`) to indicate no fit. Let's leave negative values undefined for now.